src/hooks/useShelters/useShelters.tsx

```diff
@@ -22,15 +22,20 @@ const useShelters = (options: IUseShelterOptions = {}) => {
       const headers = config.headers ?? {};
       if (cache) headers['x-app-cache'] = 'true';
       if (!append) setLoading(true);
+
+      const resolvedSearch = search ?? new URLSearchParams(window.location.search).toString()
+      const resolvedQueryParams = resolvedSearch ? new URLSearchParams(resolvedSearch) : null;
+      const order = resolvedQueryParams?.get('order') ?? 'desc';
```

Comment:
it's probably not the best way to do it, but it works. It would be nice to have a feedback on this. How should I bring `order` and `orderBy` parameters to this hook?

Reply:
You can use [use-params](https://reactrouter.com/en/main/hooks/use-params)